code/__HELPERS/dates.dm

```diff
@@ -0,0 +1,47 @@
+//Curse you calenders...
+/proc/IsLeapYear(y)
+	return ((y) % 4 == 0 && ((y) % 100 != 0 || (y) % 400 == 0))
+
+//Y, eg: 2017, 2018, 2019, in num form (not string)
+//etc. Between 1583 and 4099
+//Adapted from a free algorithm written in BASIC (https://www.assa.org.au/edm#Computer)
+/proc/EasterDate(y)
```

Comment:
make these procs on the Easter datum instead of global

Reply:
other holidays might use it

Reply:
What other holiday might use it?

Reply:
My original PR contained a few other holidays, there are atleast 2-3 that are defined as "whenever easter occurs + X" where X is some offset in days, weeks, etc.